### ISSUE TITLE
Generated type hints failed with latest mypy version (0.720)

### ISSUE THREAD

**Issue by bddppq** [pytorch/pytorch#22849]
## 🐛 Bug

mypy recently released a new version [v0.720](https://github.com/python/mypy/releases/tag/v0.720) and it fails PyTorch's generated type hints with the following error:
```
Running test_type_hints ... [2019-07-12 22:59:14.418324]
Jul 12 22:59:17 test_doc_examples (__main__.TestTypeHints) ... torch/__init__.pyi:1307: error: Name 'pstrf' already defined (possibly by an import)
Jul 12 22:59:17 FAIL
Jul 12 22:59:17 
Jul 12 22:59:17 ======================================================================
Jul 12 22:59:17 FAIL: test_doc_examples (__main__.TestTypeHints)
Jul 12 22:59:17 ----------------------------------------------------------------------
Jul 12 22:59:17 Traceback (most recent call last):
Jul 12 22:59:17   File "test_type_hints.py", line 161, in test_doc_examples
Jul 12 22:59:17     check=True)
Jul 12 22:59:17 subprocess.CalledProcessError: Command '['/opt/python/3.5/bin/python', '-mmypy', '--follow-imports', 'silent', '--check-untyped-defs', '/var/lib/jenkins/workspace/test/generated_type_hints_smoketest.py']' returned non-zero exit status 1
Jul 12 22:59:17 
Jul 12 22:59:17 During handling of the above exception, another exception occurred:
Jul 12 22:59:17 
Jul 12 22:59:17 Traceback (most recent call last):
Jul 12 22:59:17   File "test_type_hints.py", line 163, in test_doc_examples
Jul 12 22:59:17     raise AssertionError("mypy failed.  Look above this error for mypy's output.")
Jul 12 22:59:17 AssertionError: mypy failed.  Look above this error for mypy's output.
Jul 12 22:59:17 
Jul 12 22:59:17 ----------------------------------------------------------------------
```
https://circleci.com/api/v1.1/project/github/pytorch/pytorch/2185641/output/105/0?file=true
This has been caught by the CI since it installs the newest version from pypi. We have temporarily skipped this version to workaround master CI breakage.

**Comment by xuhdev** [comment]
I think #22841 has offered a proper fix... (CI passed without applying #22835)

**Comment by vishwakftw** [comment]
Yes, pstrf is being removed in #22841.

**Comment by gchanan** [comment]
@zou3519 can you check that #22841 fixes this?

**Comment by zou3519** [comment]
Confirmed fixed on master by #22841 (thanks @vishwakftw!)

**Comment by xuhdev** [comment]
Can we revert #22835 now?

**Comment by bddppq** [comment]
@zou3519 #22835 was checked in as workaround, will need to revert it to verify

**Comment by zou3519** [comment]
Oh, I see. I'll revert it.

**Comment by vishwakftw** [comment]
Closed by aeee49d51d9eb23685611213866ac33792afddea
